### DISCUSSION TITLE
Annotation method of the object detection bounding boxes

### DISCUSSION THREAD

**Discussion by mrkvgb** [bdd100k/bdd100k#241] [Q&A]
Hi thanks for your dataset! May I ask how was the ground-truth bounding box annotations for object detection rated? E.g., is it according to the rating of a group of human raters (if so, how many of them were there)? 
Thanks!

**Comment by siyuanliii** [comment]
Hi, thanks for your question. Could you explain it more? What do you mean by how annotation is rated? Do you mean how we access the quality of the annotation?

**Reply by mrkvgb** [reply]
Hi thanks for your reply! Actually I wonder how the annotations were made. I read on the BDD100K paper that they were manually annotated, but how many annotators were there? I am also curious about how the annotation quality was assessed.

**Reply by andrewliao11** [reply]
To follow up this question, I wonder what is the label specification guideline given to the annotator. Other public open datasets like [Waymo](https://github.com/waymo-research/waymo-open-dataset/blob/master/docs/labeling_specifications.md) or [nuScenes](https://github.com/nutonomy/nuscenes-devkit/blob/master/docs/instructions_nuimages.md) provide these sort-of documents
